### PR TITLE
[IMP] ecommerce: products page updated

### PR DESCRIPTION
**Pull request by Donapi**
taskid[: 4056328](https://www.odoo.com/odoo/project.task/project.task/4056328)

### CODE REVIEW

File: content/applications/websites/ecommerce/products.rst [hunk 1]
```diff
@@ -4,266 +4,341 @@
...
-- :guilabel:`Out-of-Stock (Continue Selling)`: enabling it allows customers to continue placing
-  orders even when the product is **out-of-stock**.
-  Leave it  unchecked to **prevent orders**;
-- :guilabel:`Show Available Qty`: enabling it displays the available quantity left under a specified
-  threshold on the product page. The available quantity is calculated based on the 'On hand'
-  quantity minus the quantity already reserved for outgoing transfers.
-
-.. seealso::
-   :ref:`Allow only selected customers to buy <cart/prevent-sale>`
+In the :guilabel:`Inventory Defaults` sub-section, select your eCommerce selling strategy:
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:317]:
```suggestion
Set the following fields:
```
I'm not sure they all relate to the selling strategy per say so I would try to stay a little more general (e.g., the Warehouse field, which is missing from the list of field btw)

File: content/applications/websites/ecommerce/products.rst [hunk 2]
```diff
@@ -4,266 +4,341 @@
...
---------------
-
-If you are selling non-prepackaged kits (i.e., the kits are made of individual products), we
-recommend you read the related documentation to keep track of your stock.
+- :guilabel:`Out-of-Stock`: enable :guilabel:`Continue Selling` to allow customers to continue
+  placing orders even when the product is **out-of-stock**. Leave the box unchecked to
+  **prevent orders**;
+- :guilabel:`Show Available Qty`: displays the available quantity left under a specified threshold
+  on the product page. The available quantity is calculated based on the 'On hand' quantity minus
+  the quantity already reserved for outgoing transfers.
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:324]:
```suggestion
  on the product page. The available quantity is calculated based on the :guilabel:`On hand` quantity minus
  the quantity already reserved for outgoing transfers.
```

File: content/applications/websites/ecommerce/products.rst [hunk 3]
```diff
@@ -4,266 +4,341 @@
...
-:guilabel:`Product Comparison Tool`. This tool allows to save products' **specifications** and
-compare them against each other on a single page.
-
-On the product page, scroll down to the :guilabel:`Specifications` section and click
-:guilabel:`Compare`. Repeat the same process for all products you wish to compare. Then, click the
-:guilabel:`Compare` button of the pop-up window at the bottom of the page to reach the comparison
-summary.
+Your visitors can compare products based on their attributes when the :guilabel:`Product Comparison
+Tool` is enabled. You can activate it by going to :menuselection:`Website --> Configuration -->
+Settings --> Shop - Products` section.
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:333]:
```suggestion
To allow website visitors to compare products based on their attributes, go to :menuselection:`Website --> Configuration --> Settings`, scroll down to the :guilabel:`Shop - Products` section, and enable :guilabel:`Product Comparison Tool`.
```
- non-menu items should not be in menuselection
- to simplify the sentence and avoid repetitions (is enabled/you can activate it) + respecting the order of the steps (go to XXX, find the section, enable the feature) helps make the info more readable and clear

File: content/applications/websites/ecommerce/products.rst [hunk 4]
```diff
@@ -4,266 +4,341 @@
...
 
 .. note::
    The :guilabel:`Product Comparison Tool` can only be used if :doc:`attributes <products/variants>`
-   are set on the **product's template**.
+   are set on the :ref:`product form <products/product-form>`.
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:337]: - I thought this doc page was going to be removed? which would make more sense since you talked about product variants on this doc page here? 
- I would rephrase the second part of the sentence a little to make it more clear: The :guilabel:`Product Comparison Tool` is only available for products with attributes (with a link to the relevant section on this doc).
- I know the option is activated by default when you enable the Product comparison tool, but still think it would be helpful to add a link to the Customozation options somehow in this note (using a short sentence) just in case.
-

File: content/applications/websites/ecommerce/products.rst [hunk 5]
```diff
@@ -4,266 +4,341 @@
...
+
+On your main shop page, click the :icon:`fa-exchange` (:guilabel:`Compare`) option on the products
+you want to compare. Then, click :icon:`fa-exchange` :guilabel:`Compare` in the pop-up window at the
+bottom of the page to reach the comparison summary.
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:341]: Should we also add that you can click Compare on a product page (in a tip for example)? not sure, what do you think?

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @Donapi thank you for you work on this doc page. In addition to my different comments and suggestions, here are some general remarks:
- We are implementing new guidelines for custom anchors (they're almost ready) and it would be great if we could apply them in this PR already: 1) add custom anchors above all (main) headings so they can be easily referenced later in Odoo and in the doc without having to modify this doc page later 2) use the following structure for the anchors' names: appname-filename-anchor-name (all separated by hyphens only, no slashes!), e.g. for this file: ecommerce-products-create-products. Thank you!
- It might be helpful to mention that the main Shop (products) page itself can also be customized (Customize tab in website editor) and some options might be worth explaining a little more, so maybe an additional heading above the Product page section could be added?
- No more center alignment for images
- I though you were going to remove the file /home/odoo/repositories/documentation/content/applications/websites/ecommerce/products/variants.rst since you moved the info here? 
- I haven't checked Grammarly at this point

As always, feel free to reach out if there's anything you want to discuss! Thanks :slightly_smiling_face:

File: content/applications/websites/ecommerce/products.rst [hunk 6]
```diff
@@ -4,266 +4,341 @@
...
 
-.. seealso::
-   - :doc:`products/catalog`
-   - :doc:`../../sales/sales/products_prices/products/import`
-   - :doc:`Product-related documentation <../../sales/sales>`
+From anywhere on your website, click :guilabel:`+ New`, then :guilabel:`Product`. Enter the
+:guilabel:`Product Name`, :guilabel:`Sales Price`, the default :guilabel:`Customer Taxes` for
+local transactions, and :guilabel:`Save`. You can then update the product details, add an image,
+and :ref:`customize <products/customization>` the product page. When you :guilabel:`Save`, the
+product page is automatically published.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/ecommerce/products.rst:27]: Someone who wants to create a product without making it immediately available will probably not find the information contained in this list of tips easy to find. Information on publishing a page can be found here : https://www.odoo.com/documentation/17.0/applications/websites/website/pages.html?highlight=publish#website-un-publish-page. I added a ref to that doc, to avoid duplicating the content, hope this makes sense to you :)

File: content/applications/websites/ecommerce/products.rst [hunk 7]
```diff
@@ -4,266 +4,341 @@
...
+
+.. _products/customization:
+
+Customization
+-------------
+
+To customize a product page, click :guilabel:`Edit`. Go to the :guilabel:`Blocks` tab to add
+:doc:`building blocks <../../websites/website/web_design/building_blocks>` or to the
+:guilabel:`Customize` tab to modify the page :ref:`layout <products/layout>` or add features:
+
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/ecommerce/products.rst:70]: No. Only the different options of a feature, when clarifications are useful.

File: content/applications/websites/ecommerce/products.rst [hunk 8]
```diff
@@ -4,266 +4,341 @@
...
+:guilabel:`Terms and Conditions`: toggle the switch to apply them to your product page.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/ecommerce/products.rst:71]: Regarding your third comment, I don't think it is necessary to mention it. I think people know they can click anywhere to update the page's content. We can talk about it if you want :)

File: content/applications/websites/ecommerce/products.rst [hunk 9]
```diff
@@ -4,266 +4,341 @@
...
-description are only displayed on the *product* page *(see image below)*.
+Product variants are different versions of the same product, such as various colors or materials,
+with potential differences in price and availability.
 
-.. image:: products/products-blocks.png
-   :align: center
-   :alt: Building blocks on product page
+Go to :menuselection:`Website --> Configuration --> Settings --> Shop - Products` to enable
+:guilabel:`Product Variants`. Then, go to the :guilabel:`Attributes & Variants` tab available in the
+:ref:`product forms <products/product-form>`. From there, you can add attributes and values allowing
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/ecommerce/products.rst:146]: I'm not sure of what you mean by adding a ref to the products variants doc since I remove the previous doc to only keep this one.

File: content/applications/websites/ecommerce/products.rst (hunk 1 above)
Comment (Donapi) [reply content/applications/websites/ecommerce/products.rst:317]: Updated, as discussed.

File: content/applications/websites/ecommerce/products.rst (hunk 5 above)
Comment (Donapi) [reply content/applications/websites/ecommerce/products.rst:341]: added in a note with the link to the Customization section

File: content/applications/websites/ecommerce/products.rst [hunk 10]
```diff
@@ -4,266 +4,341 @@
...
+
+Translation
+===========
+
+If multiple languages are available on your website and you want to translate a product's
+information, go to the :ref:`product form <products/product-form>` to enter the translated
+information in the related fields. Fields that support multiple languages are identifiable by their abbreviation language (e.g., EN)
+next to their field.
 
 .. image:: products/products-field-translation.png
```
[the remark this answers is not in the document]
Comment (auva-odoo) [reply content/applications/websites/ecommerce/products.rst:318]: :arrow_up:

File: content/applications/websites/ecommerce/products.rst [hunk 11]
```diff
@@ -4,275 +4,402 @@
...
 .. note::
-   Having untranslated content on a web page may be detrimental to the user experience and
-   :doc:`SEO <../../websites/website/pages/seo>`.
-.. note::
-   To check the language(s) of your website, go to :menuselection:`Website --> Configuration -->
-   Settings --> Website Info section`.
+   - Having untranslated content on a web page may be detrimental to the user experience and
+     :doc:`SEO <../../websites/website/pages/seo>`. You can use the
+     :doc:`Translate <../website/configuration/translate>` feature to allow automatic translation of
+     standard terms and provide a tool for manual content translation.
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:331]:
```suggestion
   - Having untranslated content on a web page may be detrimental to the user experience and
     :doc:`SEO <../../websites/website/pages/seo>`. You can use the
     :doc:`Translate <../website/configuration/translate>` feature to translate the page's content
```
- The last sentence sounds a bit too vague (what are *standard terms*?)
- The transition between the two parts of the last sentence (allow automatic translation xxx and provide a tool for xxx) is a bit clumsy and doesn't really flow

I tried to rephrase the sentence but actually, I think we can just simplify it since the Translations page covers the rest.

File: content/applications/websites/ecommerce/products.rst [hunk 12]
```diff
@@ -4,275 +4,402 @@
...
 
 Website availability
 --------------------
 
-A product can be set available on either *one* or *all* websites, but it is not possible to select
-*some* websites and not others. To define a product's availability, go to :menuselection:`Website
---> eCommerce --> Products`, select your product, and in the :guilabel:`Sales` tab, click the
-:guilabel:`Website` you wish the product to be available on. Leave the field empty for the products
-to be available on *all* websites.
+To set the product's availability, navigate to the :ref:`product form <ecommerce-products-product-form>`, go
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:340]: I know it's mentioned just above but still I think it'd be helpful to repeat it:
```suggestion
To set the product's website availability, navigate to the :ref:`product form <ecommerce-products-product-form>`, go
```

File: content/applications/websites/ecommerce/products.rst [hunk 13]
```diff
@@ -4,275 +4,402 @@
...
+Configuration --> Settings`, scroll down to the :guilabel:`Shop - Products` section and the
+:guilabel:`Inventory Defaults` sub-section.
 
 .. important::
-   To display the stock level on the product page, the :guilabel:`Product Type` on the **product's
-   form** must be set to :guilabel:`Storable` (only available when the **Inventory** app is
-   installed).
+   - The **Inventory** app must be installed to see the inventory management options.
+   - To display the stock level on your product page, the :guilabel:`Product Type` must be set to
+     :guilabel:`Storable` in the :ref:`product form <ecommerce-products-product-form>`.
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:361]:
```suggestion
   - To display the stock level on your product page, the :guilabel:`Product Type` must be
     :guilabel:`Storable` in the :ref:`product form <ecommerce-products-product-form>`.
```
or

```suggestion
   - To display the stock level on your product page, the :guilabel:`Product Type` field must be set to
     :guilabel:`Storable` in the :ref:`product form <ecommerce-products-product-form>`.
```

File: content/applications/websites/ecommerce/products.rst [hunk 14]
```diff
@@ -4,275 +4,402 @@
...
+.. _ecommerce-products-product-comparison:
 
 Product comparison
 ==================
 
-You can enable a **product comparison tool** for your eCommerce by going to
-:menuselection:`Website --> Configuration --> Settings --> Shop - Products`, and ticking
-:guilabel:`Product Comparison Tool`. This tool allows to save products' **specifications** and
-compare them against each other on a single page.
+To allow website visitors to compare products based on their attributes, go to
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:382]: I mentioned this in the general comment but this section is a good example of why it's best to avoid using the 2nd pronoun too much: In this case, you address the reader (i.e., in most cases, the person who configures the eCommerce app) *and* their customers (i.e. the people who will actually use the Compare tool) using *you*, which I think is a little confusing. I would actually refer customers onlu as *customers* when possible (see below for suggestions.

File: content/applications/websites/ecommerce/products.rst [hunk 15]
```diff
@@ -4,275 +4,402 @@
...
-:guilabel:`Compare` button of the pop-up window at the bottom of the page to reach the comparison
-summary.
-
-.. note::
-   The :guilabel:`Product Comparison Tool` can only be used if :doc:`attributes <products/variants>`
-   are set on the **product's template**.
+The :icon:`fa-exchange` (:guilabel:`Compare`) icon is now available on each product card on your main
+shop page. Click the :icon:`fa-exchange` (:guilabel:`Compare`) option on the products you want to
+compare. Then, click :icon:`fa-exchange` :guilabel:`Compare` in the pop-up window at the bottom of
+the page to reach the comparison summary.
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:389]:
```suggestion
The :icon:`fa-exchange` (:guilabel:`Compare`) icon is now available on each product card on the main shop page when customers hover their mouse over it. To compare products, customers can click the :icon:`fa-exchange` (:guilabel:`Compare`) option on the products they want to compare, then click :icon:`fa-exchange` :guilabel:`Compare` in the pop-up window at the bottom of the page to reach the comparison summary.
```

File: content/applications/websites/ecommerce/products.rst [hunk 16]
```diff
@@ -4,275 +4,402 @@
...
 
 .. image:: products/products-compare.png
-   :align: center
    :alt: Product comparison window
 
+.. note::
+   - The :guilabel:`Product Comparison Tool` is only available for products with
+     :ref:`attributes <ecommerce-products-product-variants>`.
+   - The :icon:`fa-exchange` (:guilabel:`Compare`) option can be specifically
+     :ref:`added or removed <ecommerce-products-customization>` on each product page.
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:398]: I'm not sure about this; it seems like the Compare option on the product page is global so you can't decide for each individual product if you want to have it or not (i.e., if you remove it, you remove it for *all product pages*).

I would suggest changing this entirely to instead mention the fact that you can you can also click Compare on a product page (like I mentioned in my first review).

File: content/applications/websites/ecommerce/products.rst [hunk 17]
```diff
@@ -4,275 +4,402 @@
...
+- :guilabel:`Datepicker`: toggle the switch to display a date range calendar to check the
+   availability of rental products over a specific period. The :guilabel:`Rental` app must be
+   installed to use this feature.
+
+- :guilabel:`Attributes`: select :guilabel:`Left` to display your product attributes on the left
+  column, and select :guilabel:`Top` to display a :icon:`fa-sliders` (:guilabel:`dropdown toggle`)
+  icon allowing users to filter products.
+
+  - :guilabel:`Price Filter`: toggle the switch to display a :guilabel:`Price Range` bar which
+    allows users to filter products according to a specific price range by dragging adjustable
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:100]:
```suggestion
    allows customers to filter products according to a specific price range by dragging adjustable
```

File: content/applications/websites/ecommerce/products.rst [hunk 18]
```diff
@@ -4,275 +4,402 @@
...
+- :guilabel:`Customers`:
+
+   - :guilabel:`Rating`: allows logged-in portal users to submit product reviews by clicking the
+     stars below the product's name. The page automatically scrolls to the
+     :guilabel:`Customer Reviews` section at the bottom, where users can share their experiences.
+     Reviews are also visible in the product's chatter in the backend. To display reviews on the
+     product page, click the :icon:`fa-plus` (:guilabel:`plus`) icon next to
+     :guilabel:`Customer Reviews`. To restrict visibility to internal employees, toggle the
+     :guilabel:`Public` switch next to the review comment.
+   - :guilabel:`Share`: adds social media and email icon buttons allowing users to share the product
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:166]:
```suggestion
   - :guilabel:`Share`: adds social media and email icon buttons allowing customers to share the product
```

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @Donapi, thank you for your work! Here's another round of comments:
- Image files: Could you please delete the image files that are not used?
- Ref names: I mentioned this last time, and I want to apologize for this again, but could you please change the names of the refs so they all match this format: app_name/file_name/anchor_name. I'm so sorry about this, this one's on me.
- Second person pronouns: I suggest reducing the number of 2nd person pronouns to what is absolutely necessary throughout this whole doc page so sentences are more neutral and focused on what needs to be done instead of the reader (e.g., in most cases, *your products* or *your shop* can be changed into *the products* or *products*, *the shop*, etc.)
- Lists: If the part after the semi-colon is a full sentence (which is the case in most of the cases here), I suggest capitalizing the first letter after the semi-colon and ending with a period instead of a semi-colon.
- In most cases, *users* are not the same as *customers* and in most cases in this doc, I think you can replace users by customers (you might want to double check though, there might be exceptions).

File: content/applications/websites/ecommerce/products.rst [hunk 19]
```diff
@@ -4,275 +4,402 @@
...
+    :guilabel:`Tags` section in the left column.
+
+- :guilabel:`Top Bar`: select :guilabel:`Sort By` to display a dropdown list in the top bar for
+  sorting products and/or :guilabel:`Layout` to display the icons for :icon:`fa-th-large`
+  (:guilabel:`Grid`) or :icon:`fa-list` (:guilabel:`List`) views.
+
+- :guilabel:`Default Sort`: choose how to sort your products: :guilabel:`Featured`,
+  :guilabel:`Newest Arrivals`, :guilabel:`Name (A-Z)`, :guilabel:`Price - Low to High` or
+  :guilabel:`Price - High to Low`.
+
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/ecommerce/products.rst:113]: I missed the Buttons option, indeed. I added them but didn't refer to the other section where they were mentioned, as it was not exactly the same description. Let me know if you want to discuss it.

**Review by auva-odoo — APPROVED** [review]
@robodoo r+
